app/scripts/store/dimStoreItem.directive.js

```diff
@@ -95,6 +110,9 @@
 
       vm.clicked = function openPopup(item, e) {
         e.stopPropagation();
+        
+        chromeStorage.drop(item.id);
+        item.isNew = false;
```

Comment:
Probably, but I thought it might get annoying if you accidentally hover over one of the new items..